swift-sdk/Internal/DependencyContainerProtocol.swift

```diff
@@ -57,7 +57,7 @@ extension DependencyContainerProtocol {
     }
 
     func createEmbeddedManager(config: IterableConfig,
-                                        apiClient: ApiClientProtocol) -> IterableEmbeddedManagerProtocol {
+                                        apiClient: ApiClientProtocol) -> IterableInternalEmbeddedManagerProtocol {
         IterableEmbeddedManager(apiClient: apiClient,
```

Comment:
Not sure why this is being changed. Can you provide a reason?

Reply:
Hi @evantk91 if you notice here I have created a new delegate EmbeddedNotifiable to sync embedded messages just like it does for syncInAppMessages, basically following the same practices like done for in-app messages. Sorry previously I forgot to push some changes from fork to this repo but now it should be good.